README.md

```diff
@@ -280,7 +280,7 @@ By contrast, BullMQ is Redis-backed and don't provide workflows, so they provide
 
 **When to use DBOS:** You need the reliability of enqueueing tasks from durable workflows.
 
-**When to use BullMQ**: You don't need durability, or you need very high throughput beyond what your Postgres server can support.
+**When to use BullMQ**: You don't need durability, you need very high throughput beyond what your Postgres server can support, or you need strictly pull-based queues (DBOS queues are push-based).
```

Comment:
I think DBOS is pull-based and BullMQ is pushed-based? Each DBOS process would poll the DB for any new tasks.

Reply:
DBOS is push-based because any work that is pushed is automatically completed, you don't need to manually "pull" any work. BullMQ is also push-based by default, but supports a strictly pull-based mode: https://docs.bullmq.io/patterns/manually-fetching-jobs

Reply:
I see, then maybe be more explicit: use BullMQ if you need to manually fetch jobs.

Reply:
@qianl15 see #1036